docs/user_guide/assessment.rst

```diff
@@ -815,6 +815,120 @@ For more examples, please
+3. Fit a distribution to this set of metric values
+4. Obtain the desired confidence intervals from that distribution
+
+Since bootstrapping involves random resampling, results will not
+be precisely reproducible between runs.
+However, as :code:`n_boot` is increased, the values found should
+converge.
+Let us look at an example:
+
+.. doctest:: assessment_metrics
+    :options:  +NORMALIZE_WHITESPACE
+
+    >>> # Enable bootstrapping with the n_boot parameter
+    >>> metric_ci = MetricFrame(metrics=skm.accuracy_score,
+    ...                         y_true=decision,
+    ...                         y_pred=prediction,
+    ...                         sensitive_features={'SF' : sensitive_feature},
+    ...                         n_boot=100)
+    >>> metric_ci.overall
+    0.4
+    >>> metric_ci.overall_ci
+    [(0.025, 0.23333333333333334), (0.975, 0.6)]
+
+We have turned on bootstrapping by setting the :code:`n_boot` parameter;
+in this case, we have requested that 100 synthetic datasets be generated
+to create the statistics.
+We see that the :attr:`MetricFrame.overall` property is unchanged.
+However, we can also access the :attr:`MetricFrame.overall_ci` property
+which contains estimates of the confidence interval.
+By default, the bootstrapping algorithm computes the 95% confidence
+interval, which ranges from the 25th millile to the 975th millile.
+The :attr:`MetricFrame.overall_ci` has returned a list of binary tuples.
+The first entry in each tuple indicates the position within the distribution
+(which are 0.025 and 0.975 for the default confidence interval), while
+the second entry in each tuple is the metric value.
+For this case, we see that we can be 95% certain that the overall
+accuracy lies between 0.23 and 0.6 (and has a nominal value of 0.4).
+We can also examine the statistics for each group, and their confidence
+intervals:
+
+.. doctest:: assessment_metrics
+    :options:  +NORMALIZE_WHITESPACE
+
+    >>> metric_ci.by_group
+    SF
+    A    0.300000
+    B    0.333333
+    C    0.625000
+    Name: accuracy_score, dtype: float64
+    >>> metric_ci.by_group_ci
+    [(0.025,
+    SF
+    A    0.000000
+    B    0.105278
+    C    0.333333
+    Name: accuracy_score, dtype: float64),
+    (0.975,
+    SF
+    A    0.621429
+    B    0.608077
+    C    0.952500
+    Name: accuracy_score, dtype: float64)]
+
+So here, we see that group 'B' has a nominal value of 0.333, with its 95% confidence
+interval running from 0.105 to 0.608.
+Finally, confidence intervals are also provided for the :method:`MetricFrame.difference`
+and :method:`MetricFrame.ratio` methods.
+For example:
+
+
+.. doctest:: assessment_metrics
+    :options:  +NORMALIZE_WHITESPACE
+
+    >>> metric_ci.difference(method='between_groups')
+    0.325
+    >>> metric_ci.difference_group_ci
+    [(0.025, 0.10214826839826842), (0.975, 0.7585227272727271)]
+
+What we see here is that the largest difference between any two of the groups
```

Comment:
Should we also mention somewhere that fairness metrics such as max-min difference are (upwards) statistically biased estimates (see Section 4 / Figure 2 of  [Lum et al. 2022](https://arxiv.org/pdf/2205.05770.pdf)), or would that make things too complicated for our user guide?